tests/schools_match_response.spec.ts

```diff
@@ -46,14 +46,21 @@ async function and_i_click_on_the_unmatched_responses_link() {
 
 async function then_i_am_on_the_unmatched_responses_page() {
   await expect(
-    p.getByRole("heading", { name: fixtures.schoolName }),
+    p.getByRole("heading", { name: "Unmatched consent responses" }),
   ).toBeVisible();
 }
 
 async function when_i_click_on_an_unmatched_response() {
-  await p
-    .getByRole("link", { name: fixtures.unmatchedConsentFormParentName })
-    .click();
+  const row = await p
+    .locator(
+      `//td[contains(., '${fixtures.unmatchedConsentFormParentName}')]/parent::tr`,
+    )
+    .first();
```

Comment:
That's a funky selector

Reply:
yeah i'm sure there's a better way of doing it, i got an assist from my good friend who works at OpenAI.